src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**IoT**
+
+* Fix #7792: IoT Hub Create is not idempotent (#7792)
+
```

Comment:
The record in `HISTORY.rst` no longer needs to be added manually. Now we have a process to automatically generate the history based on PR title

Reply:
removed